proto/klyraprotocol/yields/query.proto

```
@@ -0,0 +1,42 @@
+syntax = "proto3";
+package klyraprotocol.yields;
+
+import "gogoproto/gogo.proto";
```

Buf (1.47.2)

import "gogoproto/gogo.proto": file does not exist

(COMPILE)